docs/source/projects.rst

```diff
@@ -349,103 +349,170 @@ useful if you quickly want to test a project in your existing shell environment.
 
 .. _databricks_execution:
 
-Run a project on Databricks
+Run a Project on Databricks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Support for running projects remotely on Databricks is in public preview. To use this feature, you must have an enterprise Databricks account (Community Edition is not supported) and you must have set up the `Databricks CLI <https://github.com/databricks/databricks-cli>`_. Find more detailed instructions in the Databricks docs (`Azure Databricks <https://docs.databricks.com/applications/mlflow/index.html>`_, `Databricks on AWS <https://docs.databricks.com/applications/mlflow/index.html>`_). A brief overview of how to use the feature is as follows:
+You can run MLflow projects remotely on Databricks. To use this feature, you must have an enterprise
```

Comment:
We should be consistent how we spell MLflow Projects. In the concepts we say:

At the core, MLflow Projects

Reply:
Fixed! Thanks for catching this